### ISSUE TITLE
Edit and View Source buttons are the same

### ISSUE THREAD

**Issue by davelevine** [squidfunk/mkdocs-material#4813]
### Bug description

When clicking on either the View Source or Edit Source buttons, they both appear to bring you to the Edit page on GitHub. This can be seen in the Material for MkDocs official documentation. I noticed it specifically on the [Reference](https://squidfunk.github.io/mkdocs-material/reference/) page, but it can be seen on any page.

### Related links

The following is the page I noticed it on, but it can be seen on any page.

- [Reference](https://squidfunk.github.io/mkdocs-material/reference/)

### Reproduction

```yaml
repo_url: ...
theme:
  features:
    - content.action.edit
    - content.action.view
```

### Steps to reproduce

- Add `content.action.edit` and `content.action.view` to `mkdocs.yml`
- Run `mkdocs serve`
- Open local environment and click on either the Edit or View Source buttons.

As mentioned above, this can also be seen in the official documents.

### Browser

Firefox

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/bug-report/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) or [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/bug-report/#remove-customizations) before submitting this bug report.
- [X] I have attached a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/).

**Comment by squidfunk** [comment]
Thanks for reporting.

Note that we have new [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/bug-report/), which is also mentioned in the descriptions in the bug reporting template you filled out. You did not provide a reproduction, but since this issue is apparent from the docs, we can look into it. Please read our new bug reporting guidelines to learn how to [create a reproduction](https://squidfunk.github.io/mkdocs-material/bug-report/reproduction/).

**Comment by squidfunk** [comment]
Fixed in feef8cd6d.

So there were two things at play here. First of all, the `edit` part of the URL was not correctly replaced with `view`, which is a bug on our side. But GitHub seems to have invalidated the `view` controller, which must now be called `raw`.

**Comment by squidfunk** [comment]
Released as part of 9.0.1.
